client/me/purchases/manage-purchase/index.jsx

```diff
@@ -681,6 +685,13 @@ const ManagePurchase = React.createClass( {
 					{ contactSupportToRenewMessage }
 				</Card>
 
+				<PurchasePlanDetails
+					hasLoadedSites={ this.props.hasLoadedSites }
+					hasLoadedUserPurchasesFromServer={ this.props.hasLoadedUserPurchasesFromServer }
+					selectedSite={ this.props.selectedSite }
```

Comment:
Does it make sense to omit these props here and add them the `connect()` of `PurchasePlanDetails` directly, since it's already connected anyway? This could help make the component more independent. Or are we doing it this way for consistency with other components like `RemovePurchase` for example?